src/layouts/SidebarLayout.jsx

```
@@ -9,27 +9,35 @@ const SidebarLayout = () => {
   const location = useLocation();
   const [activeMenu, setActiveMenu] = useState(null);
 
+  // Effect to set the initial active menu but not trigger any content display
+  useEffect(() => {
+    // Find the route that contains a submenu and initialize activeMenu without affecting content
+    routes.forEach((item) => {
+      if (item.submenu && item.submenu.some(sub => location.pathname !== sub.path)) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)